Dockerfile.ci

```diff
@@ -314,6 +314,11 @@ COPY airflow/__init__.py ${AIRFLOW_SOURCES}/airflow/__init__.py
 ARG UPGRADE_TO_NEWER_DEPENDENCIES="false"
 ENV UPGRADE_TO_NEWER_DEPENDENCIES=${UPGRADE_TO_NEWER_DEPENDENCIES}
 
+# Those are additional constraints that are needed for some extras but we do not want to
+# Force them on the main Airflow package
+ARG EAGER_UPGRADE_ADDITIONAL_REQUIREMENTS
```

Comment:
Is this line necessary given the one below it?

Reply:
Yeah. I had to update it still a bit, to handle our image building process in master. Pushing a fix (without those lines)